### ISSUE TITLE
Feature request: Naming consistency

### ISSUE THREAD

**Issue by SlimeDog** [eldoriarpg/NashornJS#1]
Please use a consistent naming convention. At present we have:
- Spigot page: NashornJS Provider and CLI
- JAR: NashornPlugin-1.0.jar
- plugin.yml.name: NashornJs
- plugin subdirectory: plugins/NashornJs/ (derived from plugin.yml)

My preferences would be to use NashornJS universally:
- Spigot page: NashornJS Provider and CLI
- JAR: NashornJS-1.1.jar
- plugin.yml.name: NashornJS
- plugin subdirectory: plugins/NashornJS/

but it's your call.

**Comment by RainbowDashLabs** [comment]
Although the jar name has actually no effect on anything I renamed the artifact to `NashornJS` to change the naming of the file to the plugin name ^^

**Comment by SlimeDog** [comment]
Thank you. But internal name (plugin.yml) and therefore plugin subdirectory are still NashornJs (terminal "s").

**Comment by SlimeDog** [comment]
NashornJS 1.1
Internal version (plugin.yml) is still 1.0.

**Comment by RainbowDashLabs** [comment]
Ah now I get it apparently I wont change the plugin name. I renamed the jar name from NashornJS to NashornJs which is also correct according to the naming conventions.

I think I migrate this to gradle first... Then these shitty errors with replacements dont happen anymore. Honestly I never planed to touch this thing ever again xD

**Comment by SlimeDog** [comment]
I understand. But other plugins may decide to use this handy plugin, so you want to get it right. :)

**Comment by RainbowDashLabs** [comment]
Okay I setup the CI including the maven repository and everything else.
Nothing is burning anymore, I think.
I used PascalCase for naming which is appropriate for application naming.

Spigot page renamed to `NashornJs - Provider and CLI`
Plugin Name is now `NashornJs`
Config will be `NashornJs/config.yml`
The jar itself will be `nashornjs-version.jar`

I also moved the internal packages to nashornjs to reflect the artifact itself in the packages.

The jar name is depending on the ci. It will and should be always the name of the artifact. Artifact names should be all lowercase.
However everything is now in sync according to naming conventions.
